lib/gtfs/run.ex

```diff
@@ -1,22 +1,3 @@
 defmodule Gtfs.Run do
```

Comment:
Does this module still need to exist?

Reply:
Yeah, we use the `Run.id()` in a bunch of places.

Reply:
You don't need necessarily need to have a module to represent it. You can put `run_id` as a type in a module where it makes sense, and reference it from there.

Reply:
This function will get pulled into `run.ex` in the next PR: https://github.com/mbta/skate/pull/562#discussion_r404301474